ci/bamboo/build-osx.sh

```diff
@@ -1,32 +1,37 @@
 #!/bin/bash
+# ----------------------------------------------------------------------
+# Numenta Platform for Intelligent Computing (NuPIC)
+# Copyright (C) 2016, Numenta, Inc.  Unless you have purchased from
+# Numenta, Inc. a separate commercial license for this software code, the
+# following terms and conditions apply:
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU Affero Public License version 3 as
+# published by the Free Software Foundation.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+# See the GNU Affero Public License for more details.
+#
+# You should have received a copy of the GNU Affero Public License
+# along with this program.  If not, see http://www.gnu.org/licenses.
+#
+# http://numenta.org/licenses/
+# ----------------------------------------------------------------------
+
 set -o errexit
 set -o xtrace
 
+export ARCHFLAGS="-arch x86_64"
+
 # Fixup $PATH for --user installation
 export PATH=${HOME}/Library/Python/2.7/bin:${PATH}
 
-# Install pip
-curl --silent --show-error --retry 5 -O http://releases.numenta.org/pip/1ebd3cb7a5a3073058d0c9552ab074bd/get-pip.py
-python get-pip.py --user --ignore-installed
-
-# Install python dependencies w/ pip
-pip install \
-    --upgrade \
-    --ignore-installed \
-    --user \
-    setuptools \
-    wheel \
-    pycapnp==0.5.8 \
-    -r bindings/py/requirements.txt
-
-# Build and install nupic.core
-mkdir -p build/scripts
-cmake -DCMAKE_INSTALL_PREFIX=`pwd`/build/release -DPY_EXTENSIONS_DIR=`pwd`/bindings/py/nupic/bindings .
-make install
-./build/release/bin/cpp_region_test
-./build/release/bin/py_region_test
-./build/release/bin/unit_tests
+DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
 
-# Build wheel
-ARCHFLAGS="-arch x86_64" python setup.py bdist_wheel
-py.test bindings/py/tests
+# Build and test the manylinux wheel; see build-and-test-nupic-bindings.sh for
+# destination wheelhouse
+BUILD_TYPE="Release" \
```

Comment:
For consistency, perhaps pass `ARCHFLAGS` here as well instead of externing it at the top of the script.